myth/Api/Server/ApiController.php

```diff
@@ -186,41 +185,39 @@ public function __construct()
 			unset($file);
 		}
 
-	    parent::__construct();
```

Comment:
This was placed after the preceding code to be able to handle setting the language correctly, otherwise, IIRC it was already loaded and not loading correctly when you tried to reload with a different language. I'll have to experiment again to ensure, but I do remember running into an issue with the constructor at the top.